zerver/worker/queue_processors.py

```diff
@@ -460,6 +460,9 @@ def consume(self, data: Mapping[str, Any]) -> None:
         activate_url = do_send_confirmation_email(
             invitee, referrer, email_language, invite_expires_in_days
         )
+        if invite_expires_in_days is None:
+            # We do not queue reminder email for never expiring invitations.
+            return
```

Comment:
I am not sure whether we want to queue reminder email here. Normally we send the email two days before expiry date, but there is no expiry date for this case.